examples/practice-typescript/src/components/SideBarMenuItem/SideBarMenuItem.tsx

```diff
@@ -0,0 +1,19 @@
+import './sideBarMenuItem.scss';
+
+interface SideBarMenuItemProps {
+  text: string;
+  router: string;
+  status?: 'aside__active' | '';
+}
+
+const SideBarMenuItem = ({ text, router, status }: SideBarMenuItemProps) => {
+  return (
+    <li>
```

Comment:
SidebarMenuItem should not include `li` SidebarMenuItem can standalone

Reply:
Fixed

Reply:
@thanhnguyen-agilityio